packages/shared/package.json

```diff
@@ -43,6 +43,7 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/dompurify": "^2.3.1",
     "@types/jest": "^26.0.24",
+    "@types/node-fetch": "^2.6.11",
```

Comment:
Types was just missing here, else we'd seen this earlier already.

Reply:
Noice!